WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Welcome/MigrationWelcomeViewController.swift

```diff
@@ -125,17 +128,19 @@ final class MigrationWelcomeViewController: UIViewController {
 extension MigrationWelcomeViewController: UITableViewDataSource {
 
     func numberOfSections(in tableView: UITableView) -> Int {
-        return viewModel.blogListDataSource.numberOfSections(in: tableView)
+        1
     }
 
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return viewModel.blogListDataSource.tableView(tableView, numberOfRowsInSection: section)
+        viewModel.sites.count
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let blog = viewModel.blogListDataSource.blog(at: indexPath)
-        let cell = tableView.dequeueReusableCell(withIdentifier: MigrationWelcomeBlogTableViewCell.defaultReuseID, for: indexPath) as! MigrationWelcomeBlogTableViewCell
-        cell.update(with: blog)
+        let blog = viewModel.sites[indexPath.row]
+        let cell = tableView.dequeueReusableCell(withIdentifier: blogCellID, for: indexPath)
+        cell.contentConfiguration = UIHostingConfiguration {
```

Comment:
This is an iOS 16 API, so it requires https://github.com/wordpress-mobile/WordPress-iOS/pull/23476 to be merged first.